dockerfiles/lib/src/api/wsmaster/workspace/workspace.ts

```diff
@@ -153,13 +153,6 @@ export class Workspace {
     searchWorkspace(key:string):Promise<org.eclipse.che.api.workspace.shared.dto.WorkspaceDto> {
         Log.getLogger().debug('search workspace with key', key);
 
-        // if workspace key is too short it's a workspace name
```

Comment:
I think that by removing that part now we're unable to find a workspace by its name